pkg/server/grpc/event_data.go

```diff
@@ -77,6 +73,10 @@ func getEventValue(arg trace.Argument) (*pb.EventValue, error) {
 // based on the value type
 func parseArgument(arg trace.Argument) (*pb.EventValue, error) {
 	switch v := arg.Value.(type) {
+	case nil:
```

Comment:
Should we consider a default case as well?

Reply:
what is the default case?

Reply:
We don't return error if the type is not supported, we try to convert it to a structure, and log a message if we can't. We want to send the event even if an argument isn't supported yet: 

https://github.com/aquasecurity/tracee/blob/main/pkg/server/grpc/event_data.go#L341
https://github.com/aquasecurity/tracee/blob/main/pkg/server/grpc/event_data.go#L731

Reply:
Got it! 👍🏻